src/checkout-complete.html

```diff
@@ -7,21 +7,34 @@
 
 </head>
 <body class="main-body">
-  <div id="page_wrapper">
+  <div id="page_wrapper" class="page_wrapper">
     <div id="menu_button_container"></div>
     <div id="contents_wrapper">
       <div id="header_container" class="header_container">
-        <div class="header_label">Swag Labs</div>
+          <div class="header_label">
+              <div class="app_logo"></div>
+            </div>
         <div id="shopping_cart_container" class="shopping_cart_container"></div>
       </div>
-  
-        <div class="subheader_label">Checkout: Complete! <div class="peek"></div></div>
+      <div class="subheader">Finish</div>
       <div id="checkout_complete_container" class="checkout_complete_container">
       <h2 class="complete-header">THANK YOU FOR YOUR ORDER</h2>
       <div class="complete-text">Your order has been dispatched, and will arrive just as fast as the pony can get there!</div>
-      <img src="https://upload.wikimedia.org/wikipedia/commons/7/72/Pony_Express%2760_West_bound_1860.jpg">
-      </div>
+      <div class="pony_express"></div>
+    </div>
     </div>
+    <div class="push"></div>
   </div>
+  <footer class="footer">
+    
+      <ul class="social">
+        <li class="social_twitter">Twitter</li>
+        <li class="social_facebook">Facebook</li>
+        <li class="social_linkedin">LinkedIn</li>
+      </ul>
+     
+      <div class="footer_copy">&copy; 2018 Sauce Labs. All Rights Reserved.  Terms of Service | Privacy Policy</div>
```

Comment:
2019?

Reply:
I can fix this a couple ways. The first would be to dynamically display with vanilla js: <script>document.write(new Date().getFullYear())</script>

or we could write that in the react side, and I'd feel better if Ryan put that in. 

or we could just hard code it as 2019 and come back to in 2020 after someone notices :)

Reply:
The latter is probably good enough for now - just make sure there is a story on the backlog for the dynamic solution.